scalarm_client_csharp/Client.cs

```diff
@@ -26,6 +26,42 @@ public Client(string baseUrl)
             // this.AddHandler("application/json", new JsonConvertDeserializer());
 		}
 
+		public List<string> GetSimulationScenarioIds()
+		{
+			var request = new RestRequest("/simulation_scenarios.json", Method.GET);
+			var response = this.Execute<ResourceEnvelope<SimulationScenariosResult>>(request);
+			ValidateResponseStatus(response);
+
+			var resource = JsonConvert.DeserializeObject<SimulationScenariosResult>(response.Content);
+			if (resource.status == "ok") {
+				List<string> simulation_scenarios = resource.simulation_scenarios;
+				return simulation_scenarios;
+			} else if (resource.status == "error") {
+				throw new ScalarmResourceException<SimulationScenariosResult>(response.Data);
+			} else {
+				throw new InvalidResponseException(response);
+			}
+		}
+
+		public List<string>  GetSimulationScenarioExperiments(string scenarioId)
```

Comment:
Biorąc pod uwagę, że znam Scalarma, domyślam się że chodzi o wszystkie Eksperymenty utworzone z tego Scenariusza. Ale jeżeli ten kod jest kierowany do kogoś, kto nie zna dobrze Scalarma, to raczej nie zrozumie.